### DISCUSSION TITLE
Coping configuration from Ubuntu18 to Ubuntu22 - Android fails to login due to missing MOBIKE

### DISCUSSION THREAD

**Discussion by XploD-Ing** [strongswan/strongswan#1381] [General]
I've got strange problem:
I am upgrading my server from ubuntu 18 to ubtunu 22 (new hardware, new IP, ...)
On my old server, I got a script which was able to create all the needed certificates using ipsec pki (generating .der files) and openssl to transpose them to X.509 and PKC12.

Now on Ubuntu22, I am using the same scripts, but I am not able to connect with my mobile phone.
The log files on both servers look nearly identical, but the old one states
```
> Oct 28 08:50:29 v220191129934102946 ipsec[9574]: 15[IKE] peer supports MOBIKE
> Oct 28 08:50:29 v220191129934102946 ipsec[9574]: 15[IKE] authentication of 'C=DE, O=xxx, [CN=y@xxxde](mailto:CN=y@xxx)' 
(myself) with RSA_EMSA_PKCS1_SHA2_256 successful
```
while the other does not. Does anybody know if there's a setting I have to enable for MOBIKE? The documentation states it's on by default...

I tried to update my certificate creating skript using just ipsec pki --outform pem, but that seems to create invalid certificates.
Can somebody tell me if there's an up-to-date manual on how to create valid certificates (with ed25519 at best)?

**Comment by XploD-Ing** [comment]
Additionally, all the manuals specify a configuration file /etc/swanctl.conf
I simply installed strongswan with "apt-get install strongswan", and all I  have now is /etc/ipsec.con 
(Strongswan U5.9.5/K5.15.0-52-generic)
Are the 2 files identical, or do I have to install additional packets?

**Reply by Thermi** [reply]
install additional packages

**Reply by Thermi** [reply]
Also mobike doesn't have anything to do with the failing authentication. That's possibly a different bug on Android.
